tests/functional/api/bulk/action_test.py

```diff
@@ -7,6 +7,8 @@
 
 from h.models import Group, GroupMembership, User
 
+pytestmark = pytest.mark.usefixtures("init_elasticsearch")
```

Comment:
Adding this module-level `pytestmark` to enable the previously-autouse fixture that inits the search index only for those test modules that actually need it.